app/services/prison_visits/client.rb

```diff
@@ -75,6 +76,14 @@ def request(method, route, params:, idempotent:)
 
       response = @connection.request(options)
       JSON.parse(response.body)
+    rescue Excon::Error::Socket => e
+      if @first_time_try && !idempotent
```

Comment:
Should not that just be:
```ruby
if @first_time_try
```

As suggested [on this line](https://github.com/excon/excon/blob/9d5dc8ea4cbb217872f8a86c7c46a732972eb8bc/lib/excon/middlewares/idempotent.rb#L22) Excon won't retry idempotent request if the former request resulted in an `Excon::Error::Socket` error
Seems like retrying at least once would the desired behaviour?